lib/Icon/Icon.less

```diff
@@ -54,7 +54,7 @@
 
 .disabled .moon-icon,
 .moon-icon.disabled {
-	opacity: @moon-disabled-opacity;
-	filter: alpha(opacity=@moon-disabled-opacity-ie);
+	color: @moon-icon-font-disabled-color;
+	background-color: @moon-icon-bg-disabled-color;
```

Comment:
We'll need to continue to use the opacity property so that icon-images (`src` properties) will properly fade out too when set to disabled.